geometry_msgs/Pose start_pose
geometry_msgs/Pose goal_pose
---
std_msgs/Bool success
moveit_msgs/RobotTrajectory plan
